mriqc_group.py

```diff
@@ -38,8 +36,8 @@ def get_parser():
     return parser
 
 
-def main(argv=None):
-    args = get_parser().parse_args(argv)
+def _main(argv=None):
+    args = _get_parser().parse_args(argv)
```

Comment:
Step should be to separate CLI from workflow function, where workflow function accepts the actual arguments instead of `argv` to allow users to call from within Python.

Reply:
I'm not sure what we're going for in this, I reverted `main()`, but I think keeping `_get_parser()` private should still be alright.

Reply:
Agreed. I like making `_get_parser()` private, since it's only called by the CLI.